hat/assets/js/apps/Iaso/domains/plannings/components/LinkToPlanning.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent } from 'react';
+
+import { Link } from 'react-router';
+import { userHasPermission } from '../../users/utils';
+import { baseUrls } from '../../../constants/urls';
+import { useCurrentUser } from '../../../utils/usersUtils';
+import { useGetPlanning } from '../../assignments/hooks/requests/useGetPlanning';
+
+type Props = {
+    planningId: string | undefined;
+    planningName: string;
+};
+
+export const LinkToPlanning: FunctionComponent<Props> = ({
+    planningId,
+    planningName,
+}) => {
+    const user = useCurrentUser();
+    const { data: planning } = useGetPlanning(planningId);
```

Comment:
Not a fan of doing a request each time this link is displayed... i the team required in the url ?

Reply:
It seems necessary.  But the fetching can probably be minimized by never letting the data go stale, unless a change to the planning is made. I believe the code to update plannings already invalidates query keys, so it may just be a matter to make sure to use the right queryKey and setting the staleTime to Infinity